R/shared-methods.R

```diff
@@ -400,6 +409,10 @@ DataClass <- R6::R6Class(
 CountryDataClass <- R6::R6Class("CountryDataClass",
   inherit = DataClass,
   public = list(
+    #' @field filter_level char The level of the data to filter at. Defaults
+    #' to "1".
+    filter_level = "1",
```

Comment:
we might as well have this in data class but set to level if not otherwise initialised?